packages/superset-ui-chart-controls/src/shared-controls/components/ColumnConfigControl/ColumnConfigControl.tsx

```diff
@@ -48,8 +49,25 @@ export default function ColumnConfigControl<T extends ColumnConfig>({
   value,
   onChange,
   configFormLayout = DEFAULT_CONFIG_FORM_LAYOUT,
+  emitFilter,
   ...props
 }: ColumnConfigControlProps<T>) {
+  if (emitFilter) {
+    Object.values(configFormLayout).forEach(array_of_array => {
+      if (!array_of_array.some(arr => arr.includes('emitTarget'))) {
+        array_of_array.push(['emitTarget']);
+      }
+    });
+  } else {
+    Object.values(configFormLayout).forEach(array_of_array => {
+      console.log(array_of_array);
```

Comment:
This shouldn't be here
```suggestion
```

Reply:
Sorry about that. I should have spotted that.